cvat/apps/engine/permissions.py

```diff
@@ -259,7 +261,27 @@ def create(cls, request: ExtendedRequest, view: ViewSet, obj: Project | None, ia
         if view.basename == 'project':
             assignee_id = request.data.get('assignee_id') or request.data.get('assignee')
 
-            for scope in cls.get_scopes(request, view, obj):
+            scopes = cls.get_scopes(request, view, obj)
+
+            if cls.Scopes.UPDATE_ORGANIZATION in scopes:
+                # consider this case as deleting a project in the org A and creating a new one in the org B
+                permissions.append(cls.create_base_perm(request, view, cls.Scopes.DELETE, iam_context, obj))
+
+                if dst_org_id := request.data['organization_id']:
```

Comment:
If the field is missing in the request, accessing it will lead to the 500 error.

Reply:
This code is under `if cls.Scopes.UPDATE_ORGANIZATION in scopes` statement; scope `UPDATE_ORGANIZATION` is only added when the `organization_id ` exists in the request data. 👀